core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -4429,9 +4485,10 @@ class LogTest {
                         scheduler: Scheduler = mockTime.scheduler,
                         time: Time = mockTime,
                         maxProducerIdExpirationMs: Int = 60 * 60 * 1000,
-                        producerIdExpirationCheckIntervalMs: Int = LogManager.ProducerIdExpirationCheckIntervalMs): Log = {
+                        producerIdExpirationCheckIntervalMs: Int = LogManager.ProducerIdExpirationCheckIntervalMs,
+                        lastShutdownClean: Boolean = true): Log = {
```

Comment:
The callers of createLog() in line 652 and 2205 seem to need lastShutdownClean to be false.

Reply:
Done